src/main/java/Application.java

```diff
@@ -17,10 +14,7 @@
 @EntityScan(basePackages = {"model"})
```

Comment:
Read about @SprintBootApplication annotation, maybe, it is your case... Anyways, will be useful